apps/style/common.scss

```diff
@@ -875,6 +875,10 @@ input[type="radio"] {
   font-size: large;
 }
 
+#download-replay-video-error {
```

Comment:
I don't love adding a definition to our global scss just for this one div

Reply:
moved to inline. haha, yes, see "split up 2MB of common js/css files" in my eng swarm sprint wish list 😛 .